e2e-tests/ukef/cypress/support/portal-api/createGefFacilities.js

```diff
@@ -0,0 +1,21 @@
+const { logIn, createGefFacilities, updateGefFacilities } = require('./api');
+const { getIdFromNumberGenerator } = require('../external-api/api');
+
+module.exports = (dealId, facilities, user) => {
+  console.info('createGEFFacilities::');
```

Comment:
Looking at `insertManyGefDeals` `GEF` only has a cap `G` in the console log -- so this should probably become
```suggestion
  console.info('createGefFacilities::');
```
for easier readability